### ISSUE TITLE
Console is too far down

### ISSUE THREAD

**Issue by somePythonProgrammer** [Educational-Websites/LiveCodeEditor2#2]
Please fix the css for the output, the console is too far down.

**Comment by e-coders** [comment]
Yes!! but I didn't know How to fix the Issue. But As I've tell that press the [ F11 ] Key to toggle fullscreen. press it just once to turn on, again to turn off.

**Comment by e-coders** [comment]
Still want to fix this issue?

**Comment by somePythonProgrammer** [comment]
@e-coders  Yes, Of course I want to! Users will have trouble seeing their logs!

**Comment by e-coders** [comment]
but, I don't know how.

**Comment by Genius398** [comment]
> but, I don't know how.

fullscreen solves that issue tho

**Comment by e-coders** [comment]
hmmmm :////

**Comment by Genius398** [comment]
I would consider this resolved, but not fully

**Comment by e-coders** [comment]
umm

**Comment by somePythonProgrammer** [comment]
Let's still try to fix this, or maybe recommend the users to use the app in fullscreen in the README.

**Comment by e-coders** [comment]
Ok
